doc/tutorials/Content_Page.md

```diff
@@ -0,0 +1,86 @@
+# Content Page
```

Comment:
Maybe write `.PNG` in lowercase?

Reply:
It's annoying that very screenshot take with snipping tools autosave as `.PNG` instead of `.png`. 🤣